migrations/main.sql

```diff
@@ -241,7 +241,7 @@ CREATE TABLE IF NOT EXISTS hostel(
     street VARCHAR(100) NOT NULL,
     build VARCHAR(10) NOT NULL,
     commandant_id integer NOT NULL,
-    month_price float NOT NULL,
+    month_price decimal NOT NULL,
```

Comment:
i think we need to define limit of digits after com. So please add to decimal(4, 2)